module/config.mjs

```diff
@@ -2385,7 +2385,8 @@ DND5E.conditionTypes = {
   exhaustion: {
     label: "DND5E.ConExhaustion",
     icon: "systems/dnd5e/icons/svg/statuses/exhaustion.svg",
-    reference: "Compendium.dnd5e.rules.JournalEntry.w7eitkpD7QQTB6j0.JournalEntryPage.cspWveykstnu3Zcv"
+    reference: "Compendium.dnd5e.rules.JournalEntry.w7eitkpD7QQTB6j0.JournalEntryPage.cspWveykstnu3Zcv",
+    maximum: 6
```

Comment:
What about calling this `levels`, like "6 levels of exhaustion"? You should also update the typedef for `ConditionConfiguration` above to mention this property and note it is only used for exhaustion.

```suggestion
    levels: 6
```